rtil/src/threads/ue/rebo/livesplit.rs

```diff
@@ -97,4 +110,34 @@ impl Run {
     pub fn attempt_count() -> u32 {
         LIVESPLIT_STATE.lock().unwrap().timer.run().clone().attempt_count()
     }
+    pub fn save_splits(path: String) -> Result<(), SplitsSaveError> {
+        let state = LIVESPLIT_STATE.lock().unwrap();
+        let mut timer = state.timer.clone();
+        let filename = Path::new(&path).file_name().unwrap().to_str().unwrap().to_owned();
+        let file = match File::create(path) {
+            Ok(file) => file,
+            Err(_e) => return Err(SplitsSaveError::CreationError(format!("ERROR: Failed to create {}!", filename))),
+        };
+        let writer = BufWriter::new(file);
+        match livesplit_core::run::saver::livesplit::save_timer(&mut timer, IoWrite(writer)) {
+            Ok(_) => Ok(()),
+            Err(_e) => Err(SplitsSaveError::SaveError(format!("ERROR: Failed to save {}!", filename))),
+        }
+    }
+    pub fn load_splits(path: String) -> Result<(), SplitsLoadError> {
+        let path = Path::new(&path);
+        let filename = Path::new(&path).file_name().unwrap().to_str().unwrap().to_owned();
+        let file = match fs::read(path) {
+            Ok(file) => file,
+            Err(_e) => return Err(SplitsLoadError::OpenError(format!("ERROR: Failed to open {}!", filename))),
+        };
+        let parsed_run = match composite::parse(&file, Some(path)) {
```

Comment:
The second argument should be `None`. We don't want the lss file to be able to include / use other files. It's not necessary, as we don't support e.g. images anyway.